qiskit_ibm_runtime/options/options.py

```diff
@@ -161,6 +161,9 @@ def _get_runtime_options(options: dict) -> dict:
             if fld.name in environment:
                 out[fld.name] = environment[fld.name]
 
+        if "instance" in options:
+            out["instance"] = options["instance"]
+
```

Comment:
I believe this was another bug? Users should be able to pass the instance through options. 

```python
options = Options(instance='ibm-q/open/main')

with Session(service=service, backend=backend) as session:
    sampler = Sampler(session=session, options=options)
    job = sampler.run(bell, shots=1024)
```

Reply:
It feels to me that `instance` should be a session option and not a primitive option. I'm also not sure if the scheduler supports cross-instance session.

Reply:
`instance` is a runtime options [here](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/runtime_options.py#L34) so shouldn't it be supported in [_get_runtime_options](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/options/options.py#L151)? Or am I understand the options incorrectly?

On this branch I am able to run jobs with different instances within a single session so it seems like it's supported by the scheduler.

Reply:
will just revert this for now as it doesn't relate to the original issue